### ISSUE TITLE
Changes to dynamic screen summary due to changes to Photon Rate regions

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#275]
In https://github.com/phetsims/molecules-and-light/issues/274 we reduced the number of rates in the Photon Rate slider.

There is a dynamic phrase in the Screen Summary describing the photon rate. It would be nice to align the wording in the phrase with the actual rates.

The new rates are: very slow, slow, and fast. So, I think the change only requires changing one parameter: change "quickly" to "fast".

We already have "off", "very slowly", and "slowly".

Examples in the design doc are updated. You can find them in the Dynamic state-of-sim u=in the screen summary section of the design doc, here:
https://docs.google.com/document/d/1r9D3vq4QvmmXkQ5G6HexpybUv_vL6LrYFb3wgPfUdWg/edit#heading=h.alxa0qu6kbmj

**Comment by terracoda** [comment]
@jessegreenberg, here's an example of the prase I am talking about:
**Case 2a (light source on - 3 rates):**
- Currently, {{in observation window}}, {{infrared}} light source {{emits photons {{very slowly, slowly, fast}}}} and directly at {{carbon monoxide molecule}}.

And a further change also noted in design document is to replace the "and" in front of "directly" with a comma, like this:
- Currently, {{in observation window}}, {{infrared}} light source {{emits photons {{very slowly, slowly, fast}}}}**, directly** at {{carbon monoxide molecule}}.

Note these changes are not critical for tomorrow - #274 and timing control descriptions are the priority.

**Comment by jessegreenberg** [comment]
> The new rates are: very slow, slow, and fast. So, I think the change only requires changing one parameter: change "quickly" to "fast".

Done.

> And a further change also noted in design document is to replace the "and" in front of "directly" with a comma, like this:

This pattern doesn't work very well in the "off" case.
![image](https://user-images.githubusercontent.com/6396244/74686114-7ce0a080-519e-11ea-9aa4-e18553005935.png)

Would you still like me to change and have a custom string pattern for the off case?

**Comment by terracoda** [comment]
@jessegreenberg, I don't see "very slowly", so I might have been wrong about my comment above (sorry). I'll review more for next interview which is Friday.

**Comment by terracoda** [comment]
@jessegreenberg, yes the dynamic screen summary seems to be off by one region:

![Screen Shot 2020-02-24 at 11 17 13 AM](https://user-images.githubusercontent.com/1902116/75162432-1d334980-56f8-11ea-8c86-284058ee7b2f.png)

**Comment by terracoda** [comment]
When light source is on of one of it's 3 speeds and the sim is in Play mode, the phrase  should be:

Currently, {{in observation window}}, {{infrared}} light source {{emits photons}}
{{
- very slow 
- slowly
- fast
}}
, directly at {{carbon monoxide molecule}}.

There are other phrases. Pop them in here if you need me or @Matthew-Moore240  to look at them.

Ideally, we have 4 speeds (off, very slow, slow, and fast) that can be represented accordingly in the screen summary.

**Comment by terracoda** [comment]
@jessegreenberg, note I took the "ly" of "very slowly".

**Comment by jessegreenberg** [comment]
Over slack @terracoda summarized:
> aria-valuetext uses: off, very slow, slow, fast and screen summary uses: off, very slow, slowly and fast.

**Comment by jessegreenberg** [comment]
All done in the above commit.
